core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/datarepo/HttpDataRepoDAO.scala

```diff
@@ -3,12 +3,15 @@ package org.broadinstitute.dsde.rawls.dataaccess.datarepo
 import akka.http.scaladsl.model.headers.OAuth2BearerToken
 import bio.terra.datarepo.api.RepositoryApi
 import bio.terra.datarepo.client.ApiClient
-import bio.terra.datarepo.model.SnapshotModel
+import bio.terra.datarepo.model.{ColumnModel, SnapshotModel, TableDataType}
 
-import java.util.UUID
+import java.util.stream.Collectors
+import java.util.{Collections, UUID}
 
 class HttpDataRepoDAO(dataRepoInstanceName: String, dataRepoInstanceBasePath: String) extends DataRepoDAO {
 
+  private val datareporow_id = new ColumnModel().name("datarepo_row_id").datatype(TableDataType.STRING)
```

Comment:
Not that I do not set the `required` property, leaving it as its default, `null`.  Should this be set?  And if so, to `true` or `false`?

Reply:
I don't think there's a need to set `required`; I don't believe anything in Rawls inspects that.